packages/shared/components/inputs/Checkbox.svelte

```diff
@@ -25,7 +25,13 @@
         }}
     >
         <Icon
-            icon={checked ? `checkbox${round ? '-round' : ''}` : `checkbox-unchecked${round ? '-round' : ''}`}
+            icon={checked
```

Comment:
I would move this logic to the script section and make if a bit easier to read with if elses, its a bit hard to read with nested ? ? ?